src/handleUndo.ts

```diff
@@ -8,25 +8,27 @@ export function configureBufferSize(size: number): void {
 }
 export function handleUndo(rootReducer: ActionReducer<any>): ActionReducer<any> {
     let executedActions: Array<Action> = [];
-    let initialState = undefined;
+    let initialState = '__NO_INITIAL_STATE__';
     return (state: any, action: any) => {
+        if (initialState === '__NO_INITIAL_STATE__') {
```

Comment:
Just check for `initialState === undefined` instead of the string value.

Reply:
I was thinking about it a lot and I'm know that using this string is not so elegant.
However, `undefined` could actually be an initial state. For example:

| Action  | State  |
|---|---|
| <INITIAL_STATE>  | undefined |
|  ADD_NUMBER | [1] |
| ADD_NUMBER | [1, 2] |
| UNDO (doesn't matter which one) | [1, 2] |

This will happen, because lib will assume that `[1]` is an initial state and it will run one ADD_NUMBER action on this state instead of an `undefined`.